source/Calamari.AzureServiceFabric/Calamari.AzureServiceFabric.csproj

```diff
@@ -23,6 +23,7 @@
     SDK: https://download.microsoft.com/download/b/8/a/b8a2fb98-0ec1-41e5-be98-9d8b5abf7856/MicrosoftServiceFabricSDK.7.1.2338.msi
     -->
     <PackageReference Include="Microsoft.ServiceFabric" Version="10.1.2338" />
+    <PackageReference Include="Microsoft.ServiceFabric.Client.Http" Version="5.0.1" />
```

Comment:
if we lean into what was done in the health-check is this extra dependency required?